### PR TITLE
add rz-test '-y' option to accept all

### PR DESCRIPTION
**Pull request by b1llow**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Review by kazarmy — CHANGES_REQUESTED** [review]
Why do you want this `-y` option?

**Comment by b1llow** [comment]
> Why do you want this `-y` option?

Because sometimes changing some of rzil's code creates a large number of changes

**Comment by kazarmy** [comment]
Do you have an example of that? ... sorry not that keen on adding code that has to be manually tested and is not strictly necessary

**Comment by b1llow** [comment]
> Do you have an example of that? ... sorry not that keen on adding code that has to be manually tested and is not strictly necessary

https://github.com/rizinorg/rizin/pull/4712/commits/042f9174a4b5568975af7a4d387736c7f3452ab5

For example, in this commit, I used this feature to automatically ACCEPT over 200 asm tests at a time.

Well, if I'm the only one who uses this feature, then it's okay to turn off the PR.

**Review by XVilka — APPROVED** [review]
It's actually easy to create a test for this @kazarmy - there is `test/unit/test_rz_test.c`

File: binrz/rz-test/rz-test.c [hunk 1]
```diff
@@ -59,6 +59,7 @@ static void interact(RzTestState *state);
 static bool interact_fix(RzTestResultInfo *result, RzPVector /*<RzTestResultInfo *>*/ *fixup_results);
 static void interact_break(RzTestResultInfo *result, RzPVector /*<RzTestResultInfo *>*/ *fixup_results);
 static void interact_commands(RzTestResultInfo *result, RzPVector /*<RzTestResultInfo *>*/ *fixup_results);
+static void accept_all(RzTestState *state);
 
 static int help(bool verbose) {
```
Comment (kazarmy) [inline binrz/rz-test/rz-test.c:64]: > https://github.com/rizinorg/rizin/commit/042f9174a4b5568975af7a4d387736c7f3452ab5

> 
> For example, in this commit, I used this feature to automatically ACCEPT over 200 asm tests at a time.

Ok. Please add the flag to `rz-test -h`.

**Review by kazarmy — CHANGES_REQUESTED** [review]

**Comment by kazarmy** [comment]
> It's actually easy to create a test for this ...

Well actually it's not because whether it can be tested automatically, it's because I don't think blind fixing should be encouraged, but I suppose it's fine as long as it's used responsibly

File: binrz/rz-test/rz-test.c (hunk 1 above)
Comment (XVilka) [reply binrz/rz-test/rz-test.c:64]: @imbillow could you please address this request, then we can merge?

**Review by kazarmy — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Pull request merged**